packages/frontend/src/components/common/ResourceView/ResourceViewGrid/ResourceViewGridSpaceItem.tsx

```diff
@@ -162,13 +163,13 @@ const ResourceViewGridSpaceItem: FC<ResourceViewGridSpaceItemProps> = ({
                                 </Flex>
                             </Group>
                         </Stack>
-
                         <Box
                             sx={{
                                 flexGrow: 0,
                                 flexShrink: 0,
                                 transition: 'opacity 0.2s',
                                 opacity: hovered || opened ? 1 : 0,
+                                display: hovered || opened ? 'block' : 'none',
```

Comment:
if you add this, two lines above does not make sense and that removes the animating effect.

from the issue
> The context menu (three-dot menu) should be overlaid on top of the component, so the full name 'Marketing Dashboard' should be visible without being truncated.

I suggest making that box an `absolute` and position the same way which appears on hover (with animation like it was before) so it does not take an actual block space